logging/OpenShift.md

```diff
@@ -111,6 +111,21 @@ oc -n $LOG_NS delete route v4m-es-client-service
 ***Note:*** Do not use the `es_nodeport_enable.sh` and `es_nodeport_enable.sh` scripts 
 in an OpenShift environment.  
 
+#### Customizing Routes
+
+Routes may be configured to be host-based (default) or path-based. When
+using host-based routes, access to logging applications will be by hostname
+only - for example: `https://v4m-kibana-monitoring.apps.my-openshift-cluster.com`.
```

Comment:
I'm not sure "_...by hostname only_" is too clear.  Maybe, something like _"When using host-based routes, the name of the application is part of the hostname in the URL (for example:...).  When using path-based routes, the application name only appears as part of the path at the end of the URL"_

Reply:
I'll leave the wording to @davidbass13

Reply:
I agree with the wording suggested by @gsmith-sas, although I would add an example for the path-based URL as well.